test/test.js

```diff
@@ -98,3 +100,13 @@ describe('site', () => {
     });
   });
 });
+
+describe('TravisBuddy configuration', () => {
+  it('should parse successfully', done => {
+    configuration('bluzi/travis-buddy').then(config => {
```

Comment:
I'm not sure how you want to test it, but right now it's making a request to github for the .travis.yml. Should we use a local file instead?